src/Mapping/Driver/AbstractAnnotationDriver.php

```diff
@@ -16,7 +17,9 @@ abstract class AbstractAnnotationDriver implements AnnotationDriverInterface
     /**
      * Annotation reader instance
      *
-     * @var object
+     * @var Reader|AttributeReader|object
```

Comment:
`Reader` and `AttributeReader` are subtypes of `object`, so I think we should leave a `@todo` comment explaining what we need to do in the next major release instead of adding these types here.

Reply:
If we're going to go that route then it should be `@phpstan-param` annotated.  The entire point of adding the specific subtypes was to make clear the type of object expected.